src/player.py

```diff
@@ -1,25 +1,39 @@
+from src.game import Game
+
+
 # inherits from game class
 # this is because we need the players to access methods such as the score
 class Player(Game):
     # initialize hand with games deck
-    def __init__(self, game):
+    def __init__(self):
+        print("I compile")
 
     # lay down, should just call players hand easily
+    # put down a full set, or add to a set that is already laid down
     def lay_down(self):
+        print("I compile")
```

Comment:
you need to pass deck param

Reply:
you don't need deck parameters to lay down onto your table

Reply:
what do you think your "table" will be? How are we gonna keep track of what players have laid down and possible options for others to lay down? I think it should be a shared deck between all players

Reply:
Once something is laid down, it is irrelevant who laid it down. Thus, we can just use a general deck as long as we update scores whenever someone lays down

Reply:
What do you mean by deck? Would we just use a list of a list of each set? I get the updating scores and keeping it common, but I don't know if just a deck would work.

Reply:
we use a deck to hold cards. think of it as our array to hold numbers. We could build a subclass of deck to handle the lay down problem, idrc whatever your feeling. I think a deck would require additional methods so either workds

Reply:
@AlexFeeley

Reply:
@AlexFeeley updates?